Camel_to_snake.c

```diff
@@ -0,0 +1,36 @@
+// This program convert the CamelCase String Format to snake_case
+
+#include<stdio.h>
+#include<String.h>
+#include<stdlib.h>
+
+void camel_to_snake(char* str)
+{
+    int length=strlen(str);
+    int i=0,j=0;

```

Comment:
for this pr also. Floow the comments given in previous pr and make the changes. and let me know